### PR TITLE
PaymentRequestEvent.total should be PaymentCurrencyAmount.

### PR DESCRIPTION
**Pull request by romandev**
This PR is resolving #179 issue. As per rsolomakhin@'s suggestion, replace
PaymentItem with PaymentCurrencyAmount.

### CODE REVIEW

**Comment by romandev** [comment]
@rsolomakhin PTAL

File: index.html [hunk 1]
```diff
@@ -191,13 +191,13 @@
           <dfn data-cite=
           "!payment-request#dom-paymentmethoddata-supportedmethods">supportedMethods</dfn>,
           <dfn data-cite=
+          "!payment-request#paymentitem-dictionary">PaymentCurrencyAmount</dfn>,
```
Comment (rsolomakhin) [inline index.html]: s/paymentitem-dictionary/paymentcurrencyamount-dictionary/

**Review by rsolomakhin — CHANGES_REQUESTED** [review]
Thank you for the patch! I have a minor change request, but looks good otherwise.

File: index.html (hunk 1 above)
Comment (romandev) [reply index.html]: Good point!

Done.

**Review by rsolomakhin — APPROVED** [review]
Thank you!

**Pull request merged**